nmdc_server/migrations/versions/cf73ba2f0f85_merge_context_multiomics.py

```diff
@@ -0,0 +1,131 @@
+            multiomics_form.setdefault("dataGenerated", context_form.get("dataGenerated", None))
+            multiomics_form.setdefault(
+                "facilityGenerated", context_form.get("facilityGenerated", None)
+            )
+            multiomics_form.setdefault("facilities", context_form.get("facilities", []))
+            multiomics_form.setdefault("award", context_form.get("award", None))
+            multiomics_form.setdefault("otherAward", context_form.get("otherAward", ""))
+
+            metadata_submission.pop("contextForm", None)
+        if metadata_submission.get("multiomicsForm", None):
+            multiomics_form = metadata_submission.get("multiomicsForm", {})
+            study_form = metadata_submission.get("studyForm", {})
+
+            study_form.setdefault("alternativeNames", multiomics_form.get("alternativeNames", []))
+            study_form.setdefault("GOLDStudyId", multiomics_form.get("GOLDStudyId", ""))
+            study_form.setdefault("NCBIBioProjectId", multiomics_form.get("NCBIBioProjectId", ""))
+
+            multiomics_form.pop("alternativeNames", None)
+            multiomics_form.pop("GOLDStudyId", None)
+            multiomics_form.pop("NCBIBioProjectId", None)
+        mappings.append({"id": submission_metadata.id, "metadata_submission": metadata_submission})
+    session.bulk_update_mappings(SubmissionMetadata, mappings)
+    session.commit()
+
+
+def downgrade():
+    session = orm.Session(bind=op.get_bind())
+    mappings = []
+    for submission_metadata in session.query(SubmissionMetadata):
+        metadata_submission = submission_metadata.metadata_submission
+        if isinstance(metadata_submission, List):
+            continue
+
+        if metadata_submission.get("multiomicsForm", None):
+            multiomics_form = metadata_submission.get("multiomicsForm", {})
+            context_form = {}
+
+            context_form["datasetDoi"] = multiomics_form.get("datasetDoi", "")
+            context_form["dataGenerated"] = multiomics_form.get("dataGenerated", None)
+            context_form["facilityGenerated"] = multiomics_form.get("facilityGenerated", None)
+            context_form["facilities"] = multiomics_form.get("facilities", [])
+            context_form["award"] = multiomics_form.get("award", None)
+            context_form["otherAward"] = multiomics_form.get("otherAward", "")
+
+            metadata_submission["contextForm"] = context_form
+
+        if metadata_submission.get("studyForm", None):
+            study_form = metadata_submission.get("studyForm", {})
+            multiomics_form = metadata_submission.get("multiomicsForm", {})
+
+            multiomics_form["alternativeNames"] = study_form.get("alternativeNames", [])
+            multiomics_form["GOLDStudyId"] = study_form.get("GOLDStudyId", "")
+            multiomics_form["NCBIBioProjectId"] = study_form.get("NCBIBioProjectId", "")
+
+            metadata_submission["multiomicsForm"] = multiomics_form
```

Comment:
I believe it should be `multiOmicsForm` (capital "O"). I am seeing some issues locally when I run the migration after running `nmdc-server load-data`

Reply:
fixed in 40551fcf65e86040b72e010c54fd56f951a00041

Reply:
Confirmed by using the `load-db` command and running the migration. I can now get the list of submissions and load the multiOmicsForm.